### ISSUE TITLE
Search Input Field Above Insert Dialog

### ISSUE THREAD

**Issue by naorrosenberg** [AllenFang/react-bootstrap-table#989]
<img width="1425" alt="table" src="https://cloud.githubusercontent.com/assets/5357954/22296392/ae24fcce-e322-11e6-9c60-2d6b8c251468.png">

**Comment by AllenFang** [comment]
ok, I'll check this out and what's browser and version you use?, thank

**Comment by naorrosenberg** [comment]
```
    "react": "^15.4.1",
    "react-bootstrap-table": "^3.0.0-beta.7",
    "react-bootstrap": "^0.30.7",
    "react-dom": "^15.4.1",
    "react-router": "^3.0.1",
```

Chrome version 56.0.2924.76 (64-bit) (latest as of today)
MacOS Sierra 10.12.2 (16C67)

**Comment by AllenFang** [comment]
thanks 👍

**Comment by AllenFang** [comment]
Fixed on `v3.0.0-beta.9`, check this out 👍
